tests/integration/actions.go

```diff
@@ -538,22 +550,38 @@ type addIbcChannelAction struct {
 	portA       string
 	portB       string
 	order       string
+	// don't use configured channel version
+	// useful for non-ICS chains
+	noVersion bool
 }
 
 func (tr TestRun) addIbcChannel(
 	action addIbcChannelAction,
 	verbose bool,
 ) {
-	//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
-	cmd := exec.Command("docker", "exec", tr.containerConfig.instanceName, "hermes",
-		"create", "channel",
-		"--a-chain", string(tr.chainConfigs[action.chainA].chainId),
-		"--a-connection", "connection-"+fmt.Sprint(action.connectionA),
-		"--a-port", action.portA,
-		"--b-port", action.portB,
-		"--channel-version", tr.containerConfig.ccvVersion,
-		"--order", action.order,
-	)
+	var cmd *exec.Cmd
+	if action.noVersion {
+		//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
+		cmd = exec.Command("docker", "exec", tr.containerConfig.instanceName, "hermes",
+			"create", "channel",
+			"--a-chain", string(tr.chainConfigs[action.chainA].chainId),
+			"--a-connection", "connection-"+fmt.Sprint(action.connectionA),
+			"--a-port", action.portA,
+			"--b-port", action.portB,
+			"--order", action.order,
+		)
+	} else {
+		//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
+		cmd = exec.Command("docker", "exec", tr.containerConfig.instanceName, "hermes",
+			"create", "channel",
+			"--a-chain", string(tr.chainConfigs[action.chainA].chainId),
+			"--a-connection", "connection-"+fmt.Sprint(action.connectionA),
+			"--a-port", action.portA,
+			"--b-port", action.portB,
+			"--channel-version", tr.containerConfig.ccvVersion,
```

Comment:
Can we get rid of this version completely?

Reply:
The functionality changed, this was for adding IBC connections with sovereign chains.

I don't think this can be removed yet.